packages/components/ng-log-to-customer/src/controller.js

```diff
@@ -34,17 +34,37 @@ export default class LogToCustomerCtrl {
     this.keys = Object.keys(this.logKeys);
     this.setUrlTimeout = null;
 
-    this.setUrlSource().then(() => !this.errorMessage && this.startLog());
+    this.kinds = [];
```

Comment:
to improve readability
```suggestion
    this.logKinds = [];
```